sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice_modified.json

```diff
@@ -828,7 +828,7 @@
       "patch": {
         "summary": "Updates tags and desired properties of a device twin.",
         "description": "Updates the tags and desired properties of a device. See https://docs.microsoft.com/azure/iot-hub/iot-hub-devguide-device-twins for more information.",
-        "operationId": "Twin_UpdateDeviceTwin",
+        "operationId": "Devices_UpdateDeviceTwin",
```

Comment:
here and everywhere else, we don't need Devices/Modules a part of both the class name and method name